boards/others/esp32_devkit_v1/doit_esp32_devkit_v1_procpu.yaml

```diff
@@ -0,0 +1,22 @@
+identifier: doit_esp32_devkit_v1/esp32/procpu
+name: DOIT ESP32-Devkit-V1 PROCPU
+vendor: others
+type: mcu
+arch: xtensa
+toolchain:
+  - zephyr
+supported:
+  - adc
+  - dac
+  - gpio
+  - i2c
+  - i2s
+  - watchdog
+  - uart
+  - nvs
+  - pwm
+  - dac
```

Comment:
the `dac` is on the list twice. I suggest to remote the `dac` entry altogether to get rid of the `dac_api` issues. Or, you can add the board under the supported in that test, but I don't think it is a good idea for low profile board like this.

Reply:
ah, ok thanks I'll provide a fix, we still can submit fixes into this release window.